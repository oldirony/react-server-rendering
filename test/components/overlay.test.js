import React from 'react';
import Overlay from '../../src/components/overlay/overlay';

test('It renders correctly', ()=>{
	const Comp = ()=><div>asd</div>
	const OverlayMarkup = Overlay(Comp);

	let overlay = shallow(<OverlayMarkup />);
	expect(overlay).toMatchSnapshot();
})