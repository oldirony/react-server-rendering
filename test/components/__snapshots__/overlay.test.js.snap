// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders correctly 1`] = `
<div
  className="overlay"
>
  <div
    className="inner"
  >
    <Link
      className="close"
      replace={false}
      to="/ideas"
    >
      <svg
        className="icon"
      >
        <use
          xlinkHref="#cross"
        />
      </svg>
    </Link>
    <Comp />
  </div>
</div>
`;
